docs/posts/annotations.md

```diff
@@ -36,10 +35,7 @@ There are two options for updating draft annotations.
 
 ### Autodetect annotations csv
 
-Cellxgene can automatically find and reload your draft annotations in editable mode like so:  
-```
-annotations-directory$ cellxgene launch mydata.h5ad --experimental-annotations
-```
+Cellxgene will automatically find and reload your draft annotations in editable mode.
```

Comment:
you should describe the search path precedence.   remember that we look in multiple places if --annotations-input-file or --anotations-output-dir is not specified, which means that we may find _multiple_ annotations files.  Which is used?

Reply:
this is no longer relevant given our decision to default to a single location.   Am resolving this conversation.